app/src/main/res/layout/home_fragment.xml

```diff
@@ -1,37 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <data>
-    <import type="org.oppia.app.R"/>
-    <variable
-      name="viewModel"
-      type="org.oppia.app.home.UserAppHistoryViewModel"/>
+
+    <import type="org.oppia.app.R" />
+
     <variable
       name="presenter"
-      type="org.oppia.app.home.HomeFragmentPresenter"/>
+      type="org.oppia.app.home.HomeFragmentPresenter" />
   </data>
-  <androidx.constraintlayout.widget.ConstraintLayout
+
+  <LinearLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    android:orientation="vertical">
+
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/home_recycler_view"
```

Comment:
Because of this change StateFragmentTest file is raising issue play_exploration_button not found. Please do fix.

Reply:
There was one dummy button in this file to traverse to ExplorationActivity,

@BenHenning suggested to remove that dummy button, but because this button was connected to tests also, the test cases which were written earlier were breaking, for now I am keeping this dummy button and we will need to remove this button once all Low-fi work is done.